### PR TITLE
config_tools: update configurator document and build script

### PR DESCRIPTION
**Pull request by Weiyi-Feng**
1. update configurator document.
2. fix build script doesn't generate allchecks.xsd issue.
3. add open_devtools command, now we can get detail debug info in prod.
4. remove vconsole component, which will cause some strange js runtime issue in Linux.

Tracked-On: #6691
Signed-off-by: Weiyi Feng <weiyix.feng@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

File: misc/config_tools/configurator/README.md [hunk 1]
```diff
@@ -65,7 +80,7 @@ Run following command in the 'acrn-hypervisor' directory.
 ```shell
 cd misc/config_tools
 python scenario_config/schema_slicer.py
-python scenario_config/xs2js.py
+python scenario_config/jsnonschema/convert.py
```
Comment (dbkinder) [inline misc/config_tools/configurator/README.md:83]:
```suggestion
python scenario_config/jsonschema/convert.py
```

**Review by dbkinder — CHANGES_REQUESTED** [review]
There's a typo in a pathname.
Also, the description should provide more information about what changes are being made.

File: misc/config_tools/configurator/README.md [hunk 2]
```diff
@@ -65,7 +80,7 @@ Run following command in the 'acrn-hypervisor' directory.
 ```shell
 cd misc/config_tools
 python scenario_config/schema_slicer.py
-python scenario_config/xs2js.py
+python scenario_config/jsnoschema/convert.py
```
Comment (dbkinder) [inline misc/config_tools/configurator/README.md:83]:
```suggestion
python scenario_config/jsonschema/convert.py
```

**Review by dbkinder — CHANGES_REQUESTED** [review]
Thanks for adding better description.  There's still a typo in the path

File: misc/config_tools/configurator/README.md (hunk 2 above)
Comment (Weiyi-Feng) [reply misc/config_tools/configurator/README.md:83]: Sorry for that...add it

**Review by NanlinXie — APPROVED** [review]

**Review by dbkinder — APPROVED** [review]
LGTM thanks

**Pull request merged**